docs/PRODUCTION.md

```diff
@@ -81,44 +80,24 @@ Create a new branch off the main/master branch e.g. `git checkout -b test-env-sc
 ```
 cloud_schedulers_config = {
 
-  xgboost_training = {
-    description  = "Trigger my training pipeline in Vertex"
+  training = {
+    description  = "Trigger training pipeline in Vertex AI"
     schedule     = "0 0 1 * *"
     time_zone    = "UTC"
     template_path = "https://<GCP region>-kfp.pkg.dev/<Project ID of test project>/vertex-pipelines/xgboost-train-pipeline/v1.2"
     enable_caching = null
     pipeline_parameters = {
-      project_id = <Project ID of test project>
-      project_location = "europe-west2"
-      ingestion_project_id = <Project ID of test project>
-      model_name = "simple_xgboost"
-      model_label = "label_name"
-      dataset_id = "preprocessing"
-      dataset_location = "europe-west2"
-      ingestion_dataset_id = "chicago_taxi_trips"
-      timestamp = "2022-12-01 00:00:00"
     },
   },
 
-  xgboost_prediction = {
-    description  = "Trigger my prediction pipeline in Vertex"
+  prediction = {
+    description  = "Trigger prediction pipeline in Vertex AI"
     schedule     = "0 0 * * *"
     time_zone    = "UTC"
     template_path = "https://<GCP region>-kfp.pkg.dev/<Project ID of test project>/vertex-pipelines/xgboost-prediction-pipeline/v1.2"
     enable_caching = null
     pipeline_parameters = {
-      project_id = <Project ID of test project>
-      project_location = "europe-west2"
-      ingestion_project_id = <Project ID of test project>
-      model_name = "simple_xgboost"
-      model_label = "label_name"
-      dataset_id = "preprocessing"
-      dataset_location = "europe-west2"
-      ingestion_dataset_id = "chicago_taxi_trips"
-      timestamp = "2022-12-01 00:00:00"
-      batch_prediction_machine_type = "n1-standard-4"
-      batch_prediction_min_replicas = 3
-      batch_prediction_max_replicas = 5
+      // TODO: add all pipeline parameters which are expected by your pipeline
```

Comment:
Is there a cleaner convention here than TODO? For me TODO communicates something left not completed in _this_ codebase, rather than an activity for the user. Likely I'm being overly pedantic, so feel free to ignore.